.github/workflows/linux-wpt.yml

```diff
@@ -82,22 +82,22 @@ jobs:
             --log-raw test-wpt.${{ matrix.chunk_id }}.log \
             --always-succeed
       - name: Archive filtered results
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
         if: ${{ always() && !inputs.wpt-sync-from-upstream }}
         with:
           name: wpt-filtered-results-linux-${{ inputs.wpt-layout }}
           path: |
             filtered-test-wpt.${{ matrix.chunk_id }}.json
             unexpected-test-wpt.${{ matrix.chunk_id }}.log
       - name: Archive logs
-        uses: actions/upload-artifact@v3
+        uses: actions/upload-artifact@v4
```

Comment:
It seems upload-artifact v4  has a [breaking change](https://github.com/actions/upload-artifact/blob/main/docs/MIGRATION.md#multiple-uploads-to-the-same-named-artifact) in case of many jobs uploading to same artifact name. Is this something that affects this workflow, since we upload the chunked logs to same artifact?

Reply:
My thought exactly, but I have plan for much needed artifact refactor, currently if you run two linux jobs with different profile they overwrite packages (and wrong ones are then used in WPT runs). So I think this PR should only update all checkout actions, then I can handle artifacts refactor in separate PR.

Reply:
Oh, dang. Nice catch! I have removed the upgrade of `upload-artifact`.

It seems it will be tricky to fix this issue as I'm unsure if there is a way to avoid uploading 40 artifacts for a full build, which seems like too many.